flasksrc/schema.sql

```diff
@@ -85,3 +86,23 @@ CREATE TABLE toynet_survey_types (
   PRIMARY KEY (type_id)
   FOREIGN KEY (type_id) REFERENCES toynet_survey_questions (type_id)
 );
+
+CREATE TABLE toynet_topos (
+  topo_id INTEGER NOT NULL, --primary key
+  topology TEXT NOT NULL, --XML file (starting file for a lesson module)
+  author_id INTEGER NOT NULL,
+  PRIMARY KEY (topo_id)
+);
+
+CREATE TABLE toynet_sessions (
+  session_id INTEGER NOT NULL, --primary key
+  topo_id INTEGER NOT NULL, --links to the original topography for the lesson module (foreign key)
+  topology TEXT NOT NULL, --XML file (not the path)
+  
+  user_id INTEGER NOT NULL, --foreign key from user table
+  create_time DATETIME,
+  update_time DATETIME, 
+
+  FOREIGN KEY (topo_id) REFERENCES toynet_topos(topo_id)
+  FOREIGN KEY (user_id) REFERENCES user(id)
```

Comment:
Hm. I should merge my change first to make sure you have data in `user` to reference

Reply:
@blazebissar now I know why you mentioned merging my piece.

Update: I will push any commits necessary to make Arthur's updates work with changes after Arthur is done with all the steps.